session/src/main/scala/akka/cassandra/session/scaladsl/CassandraSession.scala

```diff
@@ -457,7 +453,7 @@ final class CassandraSession(
 /**
  * INTERNAL API
  */
-@InternalApi private[akka] final object CassandraSession {
+@InternalApi private[akka] object CassandraSession {
```

Comment:
There are a few `@InternalApi private[akka]` in `CassandraSession` that we still need from akka-persistence-cassandra. When moving to Alpakka we should consider if they should be public or we they should remain in `akka-persistence-cassandra`.